client/src/components/Login.js

```diff
@@ -55,28 +63,45 @@ function Login(props) {
           </Modal.Title>
         </Modal.Header>
         <Modal.Body>
-          <label className="text-gray">Email:</label>
-          <input
-            name="email"
-            onChange={handleEmail}
-            value={email}
-            className="form-control mb-2"
-          // placeholder="Enter email"
-          />
-          <label className="text-gray text-center">Password:</label>
-          <input
-            type="password"
-            name="password"
-            onChange={handlePassword}
-            value={password}
-            className="form-control mb-2"
-          // placeholder="Enter password"
-          />
-          <button className="btn btn-blue mb-2 mt-3 width-complete" onClick={attemptLogin} onKeyUp={(e) => {
-            if (e.keyCode === 13) { return attemptLogin }
-          }}>
-            Login
+          {showAlert ?
+
+            (<div>
+              <p className="alert-box mb-4">username or password incorrect</p>
+            </div>) : null
+          }
+          <form>
+            <label className="text-gray">Email:</label>
+            <input
+              name="email"
+              onChange={handleEmail}
+              value={email}
+              className="form-control mb-2"
+              // id="validationDefault01"
+              required
+            />
+
+            <label className="text-gray text-center">Password:</label>
+            <input
+              type="password"
+              name="password"
+              onChange={handlePassword}
+              value={password}
+              className="form-control mb-2"
+              required
+            />
+
+            <button
+              className="btn btn-blue mb-2 mt-3 width-complete"
+              onClick={(e) => email || password ? attemptLogin(e) : null}
+              onKeyUp={(e) => {
+                if (e.keyCode === 13) { return attemptLogin }
```

Comment:
I think you want to call the function here

Reply:
```suggestion
                if (e.keyCode === 13) { return email || password ? attemptLogin(e) : null }
```

To add it here for when a user hits the enter key, correct?